materialdatabase/scripts/write_wayne_kerr_permittivity_data_into_database.py

```diff
@@ -1,22 +1,22 @@
 from scipy import constants
 from materialdatabase.utils import get_closest, Z_from_amplitude_and_angle
 from materialdatabase.material_data_base_classes import *
-from materialdatabase.paths import my_wayne_kerr_measurements_path
+from materialdatabase.paths import your_sciebo
```

Comment:
this module seems to be missing?

Reply:
your_sciebo is the path to the measurement data in the sciebo-folder. i can redo that change to my_wayne_kerr_measurements_path because its more generalized then

Reply:
wasn't able to find `materialdatabase.paths` this in my local pulled code. Seems like the whole module (py-file) is missing?

Reply:
yeah, we might have to find a workaround for that. The paths.py file is individually for every user, because it determines where the measurement files are located. 
Sebastian, can you provide it in a way, that it contains the variable declarations but with None instead of the local paths?

Reply:
yes, i can fix that.